unstructured/partition/html/partition.py

```diff
@@ -70,9 +70,6 @@ def partition_html(
         The last modified date for the document.
     skip_headers_and_footers
         If True, ignores any content that is within <header> or <footer> tags
-    source_format
```

Comment:
Is this param no longer applicable or is it being pass through in **kwargs?

Reply:
Once I got the code simplified, it became apparent that no code was using that argument. It's a relevant argument elsewhere, like for specifying the format you want to convert into HTML, and it may have had a metadata role at one time, but no longer. In any case, it for sure doesn't do what is says it does so I dropped it from the docstring.